.pre-commit-config.yaml

```diff
@@ -0,0 +1,35 @@
+repos:
+# basic pre-commit
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.1.0
+    hooks:
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-added-large-files
+      - id: check-yaml
+      - id: check-toml
+      - id: detect-private-key
+# sorting imports
+  - repo: https://github.com/pycqa/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
+        args: ["--profile", "black"]
+# syntax linting and formatting
+  - repo: https://github.com/myint/autoflake
+    rev: v1.4
+    hooks:
+      - id: autoflake
+        args: [--in-place, --remove-all-unused-imports,
+               --ignore-init-module-imports]
+  - repo: https://github.com/PyCQA/flake8
+    rev: 4.0.1
+    hooks:
+      - id: flake8
+        args: [--ignore, "E203,W503",
+               --min-python-version, '3.10']
```

Comment:
Can we have 3.9 here?

Reply:
Good catch, it was from the previous commit. Fixed.